reconcile/utils/openshift_resource.py

```diff
@@ -708,5 +708,5 @@ def build_secret(
 
 def base64_encode_secret_field_value(value: str) -> Optional[str]:
     if value == "":
-        return None
+        return ""
```

Comment:
why make a specific case of this? Wouldn't the base64 call below output an empty string as well?

Reply:
Just to avoid the encode/decode for an empty string, which results in an empty string

Reply:
I think I'd just remove the test case, but it's probably a matter of personal taste :)